src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -421,23 +424,30 @@ private void sendNotifications() {
         // Update push notification devices count, which may change asynchronously
         NotificationUtils.updatePushDevices(otpUser);
 
-        if (notifications.size() == 0) {
+        boolean hasInitialReminder = initialReminderNotification != null;
+
+        if (notifications.size() == 0 && !hasInitialReminder) {
             // FIXME: Change log level
```

Comment:
Maybe out of scope, but can this `FIXME` be addressed in this PR?

Reply:
It does warrant a review of trip monitoring logs, but that should be outside of this PR.